R/github.R

```diff
@@ -7,8 +7,12 @@ github_auth <- function(token) {
 }
 
 github_response <- function(req) {
-  text <- httr::content(req, as = "text")
-  parsed <- jsonlite::fromJSON(text, simplifyVector = FALSE)
+  if (length(httr::content(req)) > 0) {
+    text <- httr::content(req, as = "text")
+    parsed <- jsonlite::fromJSON(text, simplifyVector = FALSE)
+  } else {
+    parsed <- ''
```

Comment:
Now that I've moved `github_delete_repo()` out of the package, I could actually put this function back as I found it. This was only necessary because the content of the response after successful deletion is empty and `jsonlite:;fromJSON("")` errors. But apparently that has never been a problem before.